e2e/scripts/st_caption.py

```diff
@@ -14,5 +14,46 @@
 
 import streamlit as st
 
+sidebar_markdown = """# I am a header
+
+## I am a subheader
+
+### I am a subsubheader
+
+I am some body text
+
+[I am a link](https://google.com)
+
+Foo `bar` baz"""
+
+with st.sidebar:
+    with st.beta_container():
+        st.caption(sidebar_markdown)
```

Comment:
Just making sure...does `st.sidebar.caption` not work?

Reply:
If I do `st.sidebar.caption` under a `with st.beta_container()` is it smart enough to put the beta_container inside the sidebar?

Reply:
Any reason why you need the `beta_container` in the first place?

Reply:
Oh. Good point. Originally I had more than one element in there, and I didn't want to snapshot the entire sidebar (led to complications in the past). Will remove.